iot-e2e-tests/common/src/main/java/com/microsoft/azure/sdk/iot/common/setup/provisioning/ProvisioningCommon.java

```diff
@@ -133,7 +132,54 @@ else if (attestationType == AttestationType.TPM)
         {
             throw new IllegalArgumentException("Unknown attestation type provided");
         }
+    }
 
+    public static Collection inputs() throws Exception
+    {
+        if (IntegrationTest.isPullRequest)
+        {
+            return Arrays.asList(
+                    new Object[][]
+                            {
+                                    {ProvisioningDeviceClientTransportProtocol.HTTPS, AttestationType.X509},
+                                    {ProvisioningDeviceClientTransportProtocol.AMQPS, AttestationType.X509},
+                                    {ProvisioningDeviceClientTransportProtocol.AMQPS_WS, AttestationType.X509},
+                                    {ProvisioningDeviceClientTransportProtocol.MQTT, AttestationType.X509},
+                                    {ProvisioningDeviceClientTransportProtocol.MQTT_WS, AttestationType.X509},
+
+                                    {ProvisioningDeviceClientTransportProtocol.HTTPS, AttestationType.SYMMETRIC_KEY},
+                                    {ProvisioningDeviceClientTransportProtocol.AMQPS, AttestationType.SYMMETRIC_KEY},
+                                    {ProvisioningDeviceClientTransportProtocol.AMQPS_WS, AttestationType.SYMMETRIC_KEY},
+                                    {ProvisioningDeviceClientTransportProtocol.MQTT, AttestationType.SYMMETRIC_KEY},
+                                    {ProvisioningDeviceClientTransportProtocol.MQTT_WS, AttestationType.SYMMETRIC_KEY},
+
+                                    //TODO tpm tests are flakey, so skip them for PR runs
```

Comment:
Now that we don't have separate runners for tpm/symmetrickey/x509, there is no way to just tag the TPM tests as flakey since it is in the same class as symmetric key and x509